lib/network.sh

```diff
@@ -59,34 +59,36 @@ elif [[ "${BOOT_MODE}" == "UEFISecureBoot" ]]; then
 fi
 
 # shellcheck disable=SC2155
-prefixlen PROVISIONING_CIDR "$PROVISIONING_NETWORK"
-export PROVISIONING_CIDR
-export PROVISIONING_NETMASK=${PROVISIONING_NETMASK:-$(python -c "import ipaddress; print(ipaddress.ip_network(u\"$PROVISIONING_NETWORK\").netmask)")}
+prefixlen BARE_METAL_PROVISIONER_CIDR "$BARE_METAL_PROVISIONER_NETWORK"
+export BARE_METAL_PROVISIONER_CIDR
+export BARE_METAL_PROVISIONER_NETMASK=${BARE_METAL_PROVISIONER_NETMASK:-$(python -c "import ipaddress; print(ipaddress.ip_network(u\"$BARE_METAL_PROVISIONER_NETWORK\").netmask)")}
```

Comment:
This has so many ways to fail and slip through, can we unpack it into an `if` statement please.